src/config/theme.ts

```diff
@@ -418,10 +440,31 @@ const createThemeOptions = (theme: Theme) => ({
       },
       variants: [
         {
-          props: { variant: 'gray' },
-          style: {
-            backgroundColor: '#EDEDED',
-          },
+          props: { color: 'grayscale' },
+          style: theme.unstable_sx({
+            color: 'text.primary',
+            '&:not(:disabled) .MuiButton-icon': {
+              color: theme.palette.grayscale.main,
+            },
+            '&:hover': {
+              backgroundColor: theme.palette.grayscale.p12,
+            },
+          }),
+        },
+        {
+          props: { variant: 'contained', color: 'grayscale' },
+          style: theme.unstable_sx({
+            backgroundColor: theme.palette.grayscale.p08,
+          }),
+        },
+        {
+          props: { variant: 'outlined', color: 'grayscale' },
+          style: theme.unstable_sx({
+            borderColor: theme.palette.grayscale.p30,
+            '&:hover': {
+              borderColor: theme.palette.grayscale.p30,
```

Comment:
Looks like new designs have this across the board (outlined buttons border color shouldn't change on hover), see if there is a way to configure that more generally